RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_cfi.py

```diff
@@ -1,6 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
 from RecoEgamma.EgammaIsolationAlgos.interestingEgammaIsoDetIdsSequence_cff import interestingEgammaIsoHCALSel
+from RecoJets.Configuration.CaloTowersES_cfi import *
```

Comment:
this is very suspicious to me @cms-sw/egamma-pog-l2 can you please have someone look into this ?

Reply:
test locally succeeds with only
```
import Geometry.CaloEventSetup.caloTowerConstituents_cfi
from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
CaloTowerConstituentsMapBuilder = Geometry.CaloEventSetup.caloTowerConstituents_cfi.caloTowerConstituents.clone(
    SkipHE = True
)
```